src/procedures/IssueTokens.ts

```diff
@@ -19,16 +19,28 @@ import { ModifyShareholderData } from './ModifyShareholderData';
 import { Factories } from '../Context';
 import { ZERO_ADDRESS } from '../utils/constants';
 
+/**
+ * @hidden
+ */
 export const createRefreshSecurityTokenFactoryResolver = (
   factories: Factories,
   securityTokenId: string
 ) => async () => {
   return factories.securityTokenFactory.refresh(securityTokenId);
 };
 
+/**
+ * Procedure that issue new tokens and assigns them to the shareholders list
```

Comment:
Procedure that issues tokens to the specified addresses. KYC data for those addresses must already exist or otherwise be provided in this procedure